VideoLocker/src/main/java/org/edx/mobile/player/PlayerFragment.java

```diff
@@ -750,26 +752,27 @@ private void clearAllErrors() {
 
     @Override
     public void onPrepared() {
-        if ( !isResumed() 
-                || !isVisible()) {
+        // mark prepared and allow orientation
+        isPrepared = true;
+
+        if (!isResumed() ||
+                (getParentFragment() != null && !getParentFragment().getUserVisibleHint())) {
             freezePlayer();
             return;
         }
 
+        environment.getSegment().trackVideoLoading(videoEntry.videoId, videoEntry.eid,
+                videoEntry.lmsUrl);
+
         // clear errors
         clearAllErrors();
-
-        try{
-            environment.getSegment().trackVideoLoading(videoEntry.videoId, videoEntry.eid,
-                    videoEntry.lmsUrl);
-        }catch(Exception e){
-            logger.error(e);
-        }
-
-        // mark prepared and allow orientation
-        isPrepared = true;
         initializeClosedCaptioning();
         allowSensorOrientation();
```

Comment:
nope, we should only allow orientation changes when the video has loaded and is in playable state.